test/python/circuit/test_circuit_qasm.py

```diff
@@ -54,7 +54,7 @@ def test_circuit_qasm(self):
 qreg qr2[2];
 creg cr[3];
 p(0.3) qr1[0];
-u(0.3,0.2,0.1) qr2[1];
```

Comment:
We should probably leave a test with the deprecated version, `u`, so we're sure we deprecated properly. 

Actually, this holds for all tests even though I didn't comment each one 🙂